server/api/src/rsGeneralAdmin.cpp

```diff
@@ -1135,9 +1138,29 @@ _rsGeneralAdmin( rsComm_t *rsComm, generalAdminInp_t *generalAdminInp ) {
     }
     if ( strcmp( generalAdminInp->arg0, "rm" ) == 0 ) {
         if ( strcmp( generalAdminInp->arg1, "user" ) == 0 ) {
+            const auto user_name = generalAdminInp->arg2 ? generalAdminInp->arg2 : "";
+            const auto zone_name = generalAdminInp->arg3 ? generalAdminInp->arg3 : "";
```

Comment:
Should we check for empty strings or does the `iua::server::exists` call cover that?

Reply:
From what I can tell, `iua::server::exists` just tosses the input into a genquery, which I expect to be totally fine with empty strings. Calling it end-to-end with an empty string in my docker container seems to return the expected result as well.